qtawesome/tests/test_qtawesome.py

```diff
@@ -0,0 +1,14 @@
+r"""
+Tests for QtAwesome
+"""
+# Standard library imports
+import subprocess
+
+# Test Library imports
+import pytest
+
+def test_segfault_import():
+    subprocess.call('python -c qtawesome')
```

Comment:
The most important import's for QtAwesome are on its `__init__.py`, so I don't see the need for that.